pkg/cri/config/config_unix.go

```diff
@@ -66,10 +66,11 @@ func DefaultConfig() PluginConfig {
 	tree, _ := toml.Load(defaultRuncV2Opts)
 	return PluginConfig{
 		CniConfig: CniConfig{
-			NetworkPluginBinDir:       "/opt/cni/bin",
-			NetworkPluginConfDir:      "/etc/cni/net.d",
-			NetworkPluginMaxConfNum:   1, // only one CNI plugin config file will be loaded
-			NetworkPluginConfTemplate: "",
+			NetworkPluginBinDir:        "/opt/cni/bin",
+			NetworkPluginConfDir:       "/etc/cni/net.d",
+			NetworkPluginMaxConfNum:    1, // only one CNI plugin config file will be loaded
+			NetworkPluginSetupSerially: false,
```

Comment:
I think we should also update config_windows.go

Reply:
Is this just to be explicit? As we'd end up with false regardless. Seems it's probably to be explicit as the same would hold true for `NetworkPluginConfTemplate`'s empty string

Reply:
update the config_windows.go